script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -290,12 +295,12 @@ We must have passed through Yashirogi, a suburb across the river, and entered th
 [sha:16804a1cca89d68de9ccf19feb62d20d8e4d49d6]{
 -- Page 22, Offset 1202.
 -- 路線は再び地上に向かっていく。
-The train line emerges above ground again.
+The train starts heading up, toward the surface.
 }
 [sha:fd9d45178910588d498a428406427a6a43994dc6]{
 -- Page 22, Offset 1203.
 -- ゆるやかに傾斜を昇っていく感覚。
-The sensation of gently ascending a slope.
+I sense how it is gently ascending the slope.
```

Comment:
Feels like there may be a more elegant way to phrase this one

Reply:
"I feel it gently ascending the slope."?

Reply:
Sure